Nimble/Expectation.swift

```diff
@@ -1,5 +1,25 @@
 import Foundation
 
+func expectTo<T, U where U: Matcher, U.ValueType == T>(expression: Expression<T>, matcher: U, to: String) -> (Bool, FailureMessage) {
```

Comment:
Awesome, I feel the same. First person to submit a PR to either of the CONTRIBUTING.md's with a link to the style guide we should use wins!